pkgs/tools/text/chars/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv
```

`stdenv` is never used, just the `lib` attribute for the meta section.
```suggestion
{ lib
```